coralproject/talk
# [CORL-2642]: fix load more next unread when preceding comment ignored

## What does this PR do?

Updates to use the previous key stop before the `Load more` button for new comments in oldest first. Previously, was using the last comment, but that one could sometimes be an ignored comment. 

## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags? 

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

When the `Load more` button for new comments in oldest first has an ignored comment directly preceding it, ensure that pressing Z key / next unread button clicks on load more and focuses the first unread new comment.
 
## How do we deploy this PR?
